package.json

```diff
@@ -6,10 +6,10 @@
     "npm": ">= 7.7.0"
   },
   "scripts": {
-    "prepare": "npm run clean && npm run build && npm run build --workspace=langium-vscode",
```

Comment:
Why don't we use the `prepare` script anymore?

This broke the Gitpod setup. @kaisalmen could you add the build step to gitpod.yml?

Reply:
> I removed the top-level prepare as you can't run npm install without a full rebuild. IMO it is better to have control of that: prepare is renamed to build:clean and the build pipeline is updated accordingly

That is the reason. I adjusted GH Actions, but forgot GP and noticed that yesterday when testing #910